myia/grad.py

```diff
@@ -0,0 +1,284 @@
+
+from typing import Set
+
+from .anf_ir import Apply, Constant, Parameter, Graph, ANFNode
```

Comment:
Let's use absolute imports for now, or change everything to relative imports in a single PR, but not mix the convention across files.